src/helpers/interfaces.ts

```diff
@@ -249,6 +249,11 @@ export interface Vote {
   created: number;
 }
 
+export interface VoteFilters {
+  orderDirection?: string;
+  onlyWithReason?: boolean;
```

Comment:
```suggestion
  orderDirection: string;
  onlyWithReason: boolean;
```

With one exception, every usage of this interface have both property defined. Having `undefined` as potential value is making a lot of function complain.

Reply:
Weird, could you show me what the complaints are? I don't see any..